events/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
+
+class Event(models.Model):
+    venue = models.ForeignKey(
+        'venues.Venue', models.CASCADE, related_name='events',
+    )
+    start_time = models.DateTimeField()
+    end_time = models.DateTimeField()
+    title = models.CharField(max_length=50, db_index=True)
+    description = models.TextField(blank=True)
```

Comment:
I think that there should also be a `host`, because an Event can potentially be hosted by an external organization.  Text field should suffice.